src/lightgbm/src/test/scala/VerifyLightGBMClassifier.scala

```diff
@@ -24,6 +24,8 @@ class VerifyLightGBMClassifier extends Benchmarks with EstimatorFuzzing[LightGBM
   verifyLearnerOnBinaryCsvFile("random.forest.train.csv",          "#Malignant", 1)
   verifyLearnerOnBinaryCsvFile("transfusion.csv",                  "Donated", 1)
 
+  verifySaveBooster(fileName = "PimaIndian.csv", labelColumnName = "Diabetes mellitus", outputFileName = "./model.txt")
```

Comment:
could you save this to the target directory? this way we dont rely on the directory it is run from

https://stackoverflow.com/questions/42990286/how-to-write-a-file-to-target-directory-of-scala-sbt-project

Reply:
It looks like "target/model.txt" is putting this into target folder relative to the directory it running from. Does that make sense?